pysqldb3/geopackage.py

```diff
@@ -10,14 +12,15 @@ class Geopackage:
     def __str__(self):
         pass
 
-    def __init__(self, dbo=None, path=None, table=None, schema=None, query=None, gpkg_name=None, cmd=None,
+    def __init__(self, dbo=None, path=None, table=None, schema=None, query=None, gpkg_name=None, gpkg_tbl = None, cmd=None,
```

Comment:
Most of the class variables seem too related to the methods and my or may not all be needed in all cases. the dbo param for example if you are doing geopackage to shp the db is not needed; and the table and geopackage parameters should be at the method level not the class, presumably the geopackage class is an instance for the entire geopackage which can include multiple tables and istn directly related to a database

Reply:
ok i am thinking I will make them into `@staticmethods` and add extra arguments for input for things like `write_gpkg`, `read_gpkg`, and `query_to_gpkg`